### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by codewithishu**
adding icons

### CODE REVIEW

**Comment by codewithishu** [comment]
@DhanushNehru please merge it

**Review by hasan-py — CHANGES_REQUESTED** [review]
Could you please consider to add screenshot for before and after changes on the ui? It would be great to see the visual changes.

File: index.html [hunk 1]
```diff
@@ -34,13 +36,14 @@
         }
 
         #theme-toggle {
-            margin-top: 5px;
-            padding: 10px;
+            margin-top: 15px;
```
Comment (hasan-py) [inline index.html:39]: Could we consider to use rem instead of px?

**Review by hasan-py — CHANGES_REQUESTED** [review]

**Comment by codewithishu** [comment]
Screenshot 2024-10-08 212958




Screenshot 2024-10-08 213016

File: index.html (hunk 1 above)
Comment (codewithishu) [reply index.html]: done

**Comment by codewithishu** [comment]
@DhanushNehru please merge it

**Comment by codewithishu** [comment]
@DhanushNehru please merge it sir I have already changed px to rem and uploaded screenshot of ui

**Review by hasan-py — APPROVED** [review]
LGTM, Thanks @codewithishu

File: index.html [hunk 2]
```diff
@@ -88,8 +91,8 @@
     <h1>Year Progress</h1>
     <img id="progress-img" src="https://year-in-progress.vercel.app/year-progress" alt="Year Progress">
 
-    <!-- Button positioned directly below the progress bar and aligned to the left -->
-    <button id="theme-toggle" class="btn"><i>Switch to Dark Mode</i></button>
+    <!-- Button positioned directly below the progress bar and aligned to the left --> icon
```
Comment (hasan-py) [inline index.html:94]: Why this extra text `icon` showing??  Please remove it 
Screenshot from 2024-10-09 19-02-14

File: index.html [hunk 3]
```diff
@@ -34,13 +36,14 @@
...
-            margin-top: 5px;
-            padding: 10px;
+            margin-top: 3rem;
+            padding: 0.7rem;
             cursor: pointer;
             display: inline-block;
             background-color: rgb(213, 184, 238);
             border: none;
             font-size: 1rem;
+            border-radius: 60%;
```
Comment (hasan-py) [inline index.html:46]: And your icon button is not fully circle. Its looks weird! Could you please fix this? it should be proper circle!

**Review by hasan-py — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (codewithishu) [reply index.html]: Done!

**Comment by codewithishu** [comment]
@DhanushNehru please merge it I have resolved all the problems

**Review by DhanushNehru — APPROVED** [review]
LGTM @codewithishu

**Pull request merged**